setup.py

```diff
@@ -15,7 +15,7 @@ def parse_requirements(requirements):
 
 
 PACKAGE_NAME = "PyDejavu"
-PACKAGE_VERSION = "0.1.3"
+PACKAGE_VERSION = "1.3.1"
```

Comment:
Do we need this update?

Reply:
We are calling this app in via another wrapper, we needed this to point to our latest release for it to install correctly.